### DISCUSSION TITLE
A possible way to build mobile apps for trilium

### DISCUSSION THREAD

**Discussion by sightsnow** [TriliumNext/Trilium#8625] [Ideas]
### Describe feature

I noticed the project https://github.com/nodejs-mobile/nodejs-mobile. Trilium is a nodejs project, can it help trilium build mobile applications.
The tutorial on manually installing trilium on the wiki seems to require some libraries running on liunx, is this necessary?
If none of these work, I also have plans to write a simple mobile app via etapi and Custom request handler. However, limited by the architecture of trilium, most functions may not be available, and my plan may be meaningless.
I'd like to be able to write the project though. I hope you can expose more internal APIs on etapi, such as note paths, note maps, similar notes, and note information. Maybe also provide a zen-like page for embedding inside mobile apps.
Maybe my request is a little unreasonable, please forgive me.
The above words are from Google Translate, there may be some problems, please forgive me.

**Comment by sigaloid** [comment]
The web version functions pretty close to what the nodejs-mobile would allow. https://github.com/zadam/trilium/wiki/Server-installation

**Comment by sightsnow** [comment]
> The web version functions pretty close to what the nodejs-mobile would allow. https://github.com/zadam/trilium/wiki/Server-installation

better_sqlite3 need build for android.I'm trying to build it, it may be difficult.

**Comment by meichthys** [comment]
@ApplePieC Have you had any luck getting the better_sqlite3 to build for Android?

**Comment by sightsnow** [comment]
> 

no. magic number.

**Comment by meichthys** [comment]
Issue migrated to [TriliumNext](https://github.com/TriliumNext/trilium/issues/4962) as future feature requests will be discussed and implemented there.

**Comment by meichthys** [comment]
https://github.com/FliegendeWurst/TriliumDroid

**Comment by eliandoran** [comment]
Main discussion thread: #7447
